### DISCUSSION TITLE
Collapse and expand of items in sidebar do not work correctly.

### DISCUSSION THREAD

**Discussion by vivekragunathan** [thuliteio/doks#596] [Q&A] [answered]
Except for the Prologue and Help items in the sidebar, rest of the items added do not collapse and expand correctly. I expect only one item to be expanded at a given time (the one I clicked on). However, the items expand/collapse errantly.

You can reproduce the issue in action [**here**](https://qwikref.netlify.app/docs/prologue/introduction/).

Here is a demo:

![qr](https://user-images.githubusercontent.com/292160/148035633-3848fff7-c888-4745-be88-800bdb8bb21f.gif)

**Comment by h-enk** [comment, marked as answer]
In `./config/_default/menus/menus.en.toml`, make sure to set `identifier`, like so e,g,:

```yml
[[docs]]
  name = "Bash"
  weight = 30
  identifier = "bash"
  url = "/docs/bash/"
```

And, make sure to match the (correct) identifier in the front matter of a docs page, like so, e.g.:

```md
..
menu:
  docs:
    parent: "bash"
weight: 310
toc: true
```

**Reply by vivekragunathan** [reply]
Works now beautifully .... Thanks
